.github/workflows/nodejs-demos.yml

```diff
@@ -47,6 +47,14 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
 
+    # ************** REMOVE AFTER RELEASE ********************
```

Comment:
I liked the updates you made elsewhere to change xpu to device in the actions. We should do it in this PR as well